src/components/composer/content_editable_helper.ts

```diff
@@ -128,6 +129,21 @@ export class ContentEditableHelper {
       current = it.next();
     }
 
+    if (current.value === nodeToFind) {
+      if (!current.value.hasChildNodes()) {
+        usedCharacters += nodeOffset;
+      } else {
+        const children = [...current.value.childNodes].slice(0, nodeOffset);
```

Comment:
Add a comment or a link of the [documentation](https://developer.mozilla.org/en-US/docs/Web/API/Range/endOffset), because the offset being either a number of chars, or a number of children is not intuitive _at all_